data.sql

```diff
@@ -25,3 +25,39 @@ UPDATE animals SET owner_id =
 		ELSE (SELECT id FROM owners WHERE full_name = 'Dean Winchester')
 	END
 COMMIT
+
+-----------------------------------------------------------------------------
+
+INSERT INTO VETS (NAME, AGE, date_of_graduation) 
+VALUES ('William Tatcher', 45, '4/23/2000'), ('Maisy Smith', 26, '1/17/2019'), ('Stephanie Mendez', 64, '5/4/1981'), ('Jack Harkness', 38, '6/8/2008')
+
+INSERT INTO specializations (species_id, vets_id) 
+	SELECT s.id, v.id FROM vets v, species s JOIN animals a ON s.id = a.id WHERE s.name = 'Pokemon' AND v.name = 'William Tatcher'
+INSERT INTO specializations (species_id, vets_id) 
+	(SELECT s.id, v.id FROM vets v, species s JOIN animals a ON s.id = a.id WHERE s.name IN ('Digimon', 'Pokemon') AND v.name = 'Stephanie Mendez')
+INSERT INTO specializations (species_id, vets_id) 
+	SELECT s.id, v.id FROM vets v, species s JOIN animals a ON s.id = a.id WHERE s.name = 'Digimon' AND v.name = 'Jack Harkness'
+		
```

Comment:
- Kindly make Sure to add one more insertion to the `specialiazation` table that's missing, in you `data.sql`. There should be 4 insertions but you only have 3. Kindly add that.

Reply:
@GedeonTS  thank you for your constructive feedback. On this particular comment, have you seen this one? The second insertion statement adds two records at a time. I appreciate to if you consider it.
 
![image](https://github.com/Microverse-Fullstack-Program/vet-clinic-database/assets/73167960/6743a0f9-ef20-402b-8156-1171e042bf9b)